### ISSUE TITLE
The curiefense image uploaded on August 31 has a serious bug and cannot be used normally

### ISSUE THREAD

**Issue by houhuijian** [curiefense/curiefense#887]
After the new version is upgraded, the initialization rule has no action, and the action cannot be saved after selecting the action. The newly created rule can be saved, but it does not take effect. The rules that have been tested so far include that the tag only of the global filter does not take effect, and the flowcontrol configuration does not take effect. In the log An error occurs, when resolving entry from /cf-config/current/config/json/globalfilter-lists.json:invalid type:map,expected a string
when resolving entry from /cf-config/current/config/json/flow-control.json: missing field 'tags'

**Comment by tzuryby** [comment]
that is correct. 1.5.0 is still the official release. a new one is coming within 2-3 weeks.
